src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -732,58 +732,11 @@ void QgsRasterLayerProperties::sync()
     mInvertColorsCheck->setChecked( hueSaturationFilter->invertColors() );
   }
 
-  /*
```

Comment:
Why is all this code redundant? I can't see where the replacement is? (unless it's coming from QgsRasterLayerProperties::setRendererWidget)

Reply:
QgsRasterTransparencyWidget::syncToLayer() is called from QgsRasterTransparencyWidget ctor which is called from QgsRasterLayerProperties::QgsRasterLayerProperties.

I think that the code was redundant because the old implementation was not using the widget itself, this PR is a followup of https://github.com/qgis/QGIS/pull/45063